lib/hanami/slice_registrar.rb

```diff
@@ -46,19 +46,16 @@ def freeze
     end
 
     def load_slices
-      slice_configs = Dir[root.join(CONFIG_DIR, SLICES_DIR, "*#{RB_EXT}")]
-        .map { |file| File.basename(file, RB_EXT) }
+      slice_configs = root.join(CONFIG_DIR, SLICES_DIR).glob("*#{RB_EXT}")
```

Comment:
TIL `Pathname#glob`. Seriously, it has to be one of the best Ruby classes out there!